rosdep/ruby.yaml

```diff
@@ -86,6 +86,11 @@ ruby:
     xenial: [ruby, ruby-dev]
     yakkety: [ruby, ruby-dev]
     zesty: [ruby, ruby-dev]
+ruby-backports:
+  arch: [ruby-backports]
```

Comment:
```suggestion
```

AUR packages should not be added to Arch Linux directly: https://github.com/ros/rosdistro/blob/master/CONTRIBUTING.md#arch-linux

Reply:
Done. I noticed that `ruby-nokogiri` and `ruby-facets` are not core packages either:
- https://www.archlinux.org/packages/?q=ruby-nokogiri
- https://www.archlinux.org/packages/?q=ruby-facets

Do you want me to remove those entries, too?

Reply:
If you do please do so in a separate PR for easy review. One of the ongoing challenges with maintaining the rosdep definitions for Arch is that packages can move between the AUR and community repositories based on Trusted User adoption or orphaning.